src/card/index.d.ts

```diff
@@ -15,7 +15,7 @@ export interface CardProps {
   readonly action?: React.ReactNode;
   readonly children?: React.ReactNode;
   readonly hasThumbnail?: (props: {readonly thumbnail?: string}) => boolean;
-  readonly headerImage?: string;
+  readonly headerImage?: string | any;
```

Comment:
can we create a type for the object here?

Reply:
of course, I will create it asap